src/latex.jl

```diff
@@ -3,8 +3,11 @@
 include("latexsvgfile.jl")
 latex(text::LaTeXString) = latex(text, O)
 latex(text::LaTeXString, pos::Point) = latex(text, pos, :stroke)
+latex(text::LaTeXString, pos::Point, valign::Symbol, halign::Symbol) =
+    latex(text, pos, :stroke, valign = valign, halign = halign)
 latex(text::LaTeXString, x, y) = latex(text, Point(x, y), :stroke)
-
+latex(text::LaTeXString, x, y, valign::Symbol, halign::Symbol) =
+    latex(text, Point(x, y), :stroke, valign = valign, halign = halign)
 """
     latex(text::LaTeXString, pos::Point, object::Symbol)
```

Comment:
this is missing the updated function signature.
Needs also explanation of the arguments later and the difference in positional and keyword arguments there as you have defined them above.